presto-array/src/main/java/com/facebook/presto/array/ShortBigArray.java

```diff
@@ -0,0 +1,147 @@
+/*
```

Comment:
@rash67 : I copied from LongBigArray since we need memory tracking as well :). Actually I note `sizeOfLongArray` has also to be changed . Will do this :)

But shouldn't need too much time on this :)